src/scss/App.scss

```diff
@@ -1,31 +1,9 @@
-$background-color-main: #F4F4F4;
-$background-color-purple: #D0073E;
-$background-color-grey: #EDEDED;
-$text-dark-grey: #919191;
-$border-shadow: #D4D3D3;
-$border-dark-grey: #919191;
-$grey-three: #919191;
-$white: #FFFFFF;
+@import '../components/header.module.scss';
+@import '../components/filters.module.scss';
+@import '../components/players/playersList.module.scss';
+@import '../components/players/playerDetails.module.scss';
```

Comment:
Can you add a `TO DO` comment here. Just a reminder that these imports are temporary.

Reply:
I've added the comment. What will eventually happen with these imports, just get deleted?

Reply:
yup, they'll get removed

Reply:
once css modules is completed throughout the app